src/lib/y2network/config_writer/sysconfig.rb

```diff
@@ -97,6 +96,43 @@ def write_ipv6_forwarding(forward_ipv6)
 
         Yast::SCR.Execute(Yast::Path.new(".target.bash"), "/usr/sbin/sysctl -w #{IPV6_SYSCTL}=#{sysctl_val.shellescape}") == 0
       end
+
+      # Finds routes for a given interface
+      #
+      # @param iface  [Interface,:any] Interface to search routes for
+      # @param routes [Array<Route>] List of routes to search in
+      # @return [Array<Route>] List of routes for the given interface
+      #
+      # @see #find_routes_for_any
+      # @see #find_routes_for_iface
+      def find_routes_for(iface, routes)
+        iface == :any ? find_routes_for_any(routes) : find_routes_for_iface(iface, routes)
+      end
+
+      # Finds routes which are not tied to an specific interface
+      def find_routes_for_any(routes)
+        routes.select { |r| r.interface == :any }
+      end
+
+      # Finds routes for a given interface
+      #
+      # @param iface  [Interface,:any] Interface to search routes for
+      # @param routes [Array<Route>] List of routes to search in
+      # @return [Array<Route>] List of routes for the given interface
+      def find_routes_for_iface(iface, routes)
+        routes.select do |route|
+          route.interface != :any && route.interface.name == iface.name
+        end
+      end
+
+      # Returns the routes file for a given interace
+      #
+      # @param iface  [Interface,:any] Interface to search routes for
+      # @return [Y2Network::SysconfigRoutesFile]
+      def routes_file_for(iface)
+        return Y2Network::SysconfigRoutesFile.new if iface == :any
+        Y2Network::SysconfigRoutesFile.new("/etc/sysconfig/network/ifroute-#{iface.name}")
```

Comment:
not, sure, but can it make sense to have such path in Interface object? like iface.routing_file_path?

Reply:
Probably yes. We should come up with a solution for the ":any everywhere" problem and this one together.